src/routes/login/index.svelte

```diff
@@ -13,23 +13,29 @@ export async function load({ session }) {
 </script>
 
 <script>
-import GitHub from '@svicons/fa-brands/github.svelte'
-import { oAuthLogin } from '$lib/utils/magic'
-import Footer from '$lib/components/Footer.svelte'
 import Logo from '$components/Logo.svelte'
+import Footer from '$lib/components/Footer.svelte'
+import { oAuthLogin } from '$lib/utils/magic'
+import GitHub from '@svicons/fa-brands/github.svelte'
 let error = false
+let bgImages = ['https://images.unsplash.com/photo-1653856114616-ec604cb38719?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=1770&q=80',
+'https://images.unsplash.com/photo-1645956949733-27efb4bf8f53?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=1770&q=80',
+'https://images.unsplash.com/photo-1653856114603-d67a3735c376?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=1770&q=80',
+'https://images.unsplash.com/photo-1653856289645-8601ab0e422f?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1yZWxhdGVkfDR8fHxlbnwwfHx8fA%3D%3D&auto=format&fit=crop&w=1770&q=60'
+]
+let bgImage = bgImages[Math.floor(Math.random() * bgImages.length)];
 </script>
 
 <svelte:head>
   <title>Login</title>
 </svelte:head>
-<div class="flex justify-center items-center h-full">
+<div class="flex justify-center items-center h-full" style="background-image: url('{bgImage} ');">
```

Comment:
Hmmm. Is there a reason `{bgImage} ` has a space after it? Seems like it's not breaking though

Reply:
I won't break anything also I guess we can add no-repeat, position etc together

Reply:
Haha just checked. Yeah my Discord notifications for servers are all muted except for ones that I'm active in